overviews/core/_posts/2012-09-20-futures.md

```diff
@@ -36,6 +36,134 @@ in the case of an application which requires blocking futures, for an underlying
 environment to resize itself if necessary to guarantee progress.
 -->
 
+A typical future look like this:
+
+    val inverseFuture : Future[Matrix] = Future {
```

Comment:
We normally write the type ascription immediately after the identifier, without the space:

    val inverseFuture: Future[Matrix] = ...